pom.xml

```diff
@@ -2606,6 +2606,35 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>3.1.3</version>
+        <configuration>
+          <classFilesDirectory>${basedir}/target/scala-2.11/classes</classFilesDirectory>
+          <testClassFilesDirectory>${basedir}/target/scala-2.11/test-classes</testClassFilesDirectory>
+          <effort>Max</effort>
+          <threshold>Low</threshold>
+          <xmlOutput>true</xmlOutput>
+          <visitors>FindPuzzlers</visitors>
+          <fork>false</fork>
+        </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>com.github.spotbugs</groupId>
+            <artifactId>spotbugs</artifactId>
+            <version>3.1.3</version>
```

Comment:
Does the spotbugs 3.1.3 plugin not already have this dependency? I would have assumed that's the default and we don't want to manage it separately. If we do maybe make a property for this version to make sure it doesn't vary?

Reply:
Sure, let us drop this part.